### PR TITLE
View tasks

### PR DESCRIPTION
**Pull request by Wade-Coplen**
# Description
- added onClick to the /dashboard 
- created a Modal to link to that onClick
- users can now click on a task and view details of task
- mild styling changes to said View Task form
- all tests passed
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x ] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [x ] Test A

# Checklist

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My code has been reviewed by at least one peer

- [x] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x ] There are no merge conflicts

### CODE REVIEW

File: src/components/ViewTask.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    marginTop: '5px',
+  };
+  const categoryName = {
+    paddingLeft: '35px',
+  };
+
+  return (
+    <div>
+      <h1>{task.task_name}</h1>
+      <div style={h3Div}>
```
Comment (jgarrow) [inline src/components/ViewTask.js:28]: Don't forget to use `sx` instead of `style` ;)

Comment (Wade-Coplen) [reply src/components/ViewTask.js:28]: Will do. Thanks for the reminder.

**Comment by acarrillo3** [comment]
I resolved the conflict at app.js file by accepting incoming changes.

You have to 
git pull origin view_tasks
git add .
git commit -m "up to date with view tasks branch"
git pull origin  development"
npm install    /*** so that it can install new dependencies material UI  ***/
fix conflicts by accepting incoming changes or accepting the current changes (changes you made)
git commit -m "up to date with development "
git push origin view_task

**Comment by jgarrow** [comment]
Also, that test is failing 'cause the `TaskPage.js` file got deleted or renamed previously. But it's still being imported and used in `App.js`. So the test is freaking out since the file doesn't exist

**Pull request merged**